web/pwa/screens/BeforeWork.js

```diff
@@ -194,6 +203,44 @@ export function BeforeWork({ beginNewMission, openHistory, missions }) {
     ["desc"]
   );
 
+  function shouldDisplayEmployeeSocialImpactSurvey() {
+    const surveyId = process.env.REACT_APP_SURVEY_EMPLOYEE_SOCIAL_IMPACT;
+    if (!surveyId) {
+      return false;
+    } else if (companies?.some(c => c.admin)) {
+      return false;
+    } else if (
+      !isDateBeforeNbDays(unixTimestampToDate(userInfo.creationTime), 61)
```

Comment:
:green_circle: On peut supprimer le magic number avec une constante : 
```suggestion
      !isDateBeforeNbDays(unixTimestampToDate(userInfo.creationTime), NB_DAYS_BEFORE_FIRST_DISPLAY)
```